frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx

```diff
@@ -0,0 +1,94 @@
+import * as classNames from 'classnames';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  YellowExclamationTriangleIcon,
+  GreenCheckCircleIcon,
+  BlueInfoCircleIcon,
+  RedExclamationCircleIcon,
+} from '@console/shared';
+import { ArrowCircleUpIcon } from '@patternfly/react-icons';
+import { global_info_color_100 as blueInfoColor } from '@patternfly/react-tokens';
+import { history, Timestamp } from '@console/internal/components/utils';
+
+import { NotificationTypes } from './notification-drawer';
+
+const NotificationIcon: React.FC<NotificationIconTypes> = ({ type }) => {
+  switch (type) {
+    case NotificationTypes.update:
+      return <ArrowCircleUpIcon color={blueInfoColor.value} />;
+    case NotificationTypes.success:
+      return <GreenCheckCircleIcon />;
+    case NotificationTypes.critical:
+      return <RedExclamationCircleIcon />;
+    case NotificationTypes.info:
+      return <BlueInfoCircleIcon />;
+    case NotificationTypes.warning:
+    default:
+      return <YellowExclamationTriangleIcon />;
+  }
+};
+
+const NotificationEntry: React.FC<NotificationEntryProps> = ({
+  title,
+  description,
+  isRead = false,
+  timestamp,
+  targetURL,
+  toggleNotificationDrawer,
+  type,
+}) => {
+  const notificationAction = type === NotificationTypes.update && targetURL && (
+    <div className="pf-c-notification-drawer__header-action">
+      <Link to="">Install Update</Link>
```

Comment:
Empty `to=""`.  Guessing is a `todo`.

Reply:
@benjaminapetersen this was in the initial design where we would use the Install Update right from the notification drawer, but the design has changed here (https://github.com/openshift/openshift-origin-design/pull/306) where the update notifications now just direct the user to the cluster updates page. @cshinn updated this last week.

Reply:
This works with `<Link to="">` empty tho?